Provider Provisioner/src/handle-input-offer.ts

```diff
@@ -28,19 +35,49 @@ export async function handleInputOffer(params: HandleInputOfferParams): Promise<
   if (!orders.length) {
     return;
   }
+  await spctlService.initialize();
 
   const resourceJsonPath = 'resource.json';
   await fs.writeFile(resourceJsonPath, JSON.stringify(inputOffer.resourceFileContent), 'utf-8');
 
+  await Promise.allSettled(
+    orders.map(async (order) => {
+      try {
+        await spctlService.completeOrder({
+          orderId: order.id,
+          status: OrderStatus.Done,
+          resultPath: resourceJsonPath,
+        });
+      } catch (err) {
+        if (err instanceof StorageResourceValidationError) {
+          log.error({ err }, `Completing order ${order.id} with status Error`);
+          return await completeWithError({
+            orderId: order.id,
+            errorMessage: err.message,
+            spctlService,
+          }).catch((err) => log.error({ err }, 'Completing order with Error failed. Ignoring...'));
+        }
+
+        log.error({ err }, `Failed to complete order ${order.id}. Ignoring..`);
```

Comment:
```suggestion
        log.error({ err }, `Failed to complete order ${order.id} with status Done`);
```
Please move this log after 51 line.